tests/kokkos/workspace_tests.cpp

```diff
@@ -87,31 +87,33 @@ static void unittest_workspace()
 
   int nerr = 0;
   const int ints_per_ws = 37;
-  static constexpr const int num_ws = 4;
+  static constexpr const int num_workspaces = 4;
   const int ni = 128;
   const int nk = 128;
 
   TeamPolicy policy(ExeSpaceUtils<ExeSpace>::get_default_team_policy(ni, nk));
 
   {
-    WorkspaceManager<double, Device> wsmd(17, num_ws, policy);
+    const int slot_length = 17;
+    WorkspaceManager<double, Device> wsmd(slot_length, num_workspaces, policy);
```

Comment:
Sorry for being pedantic, but is the 2nd ctor arg the number of workspaces or rather the number of slots in a workspace? I thoguht the latter, since the number of workspaces depends on the number of concurrent teams, which is deduced from the policy...

Reply:
I see what you are saying, yeah it should be `n_slots`, each with `slot_length=17`. Maybe the original `num_ws` stood for "Number of Workspace Slots"?

Reply:
Should it be something like `num_slots_per_team`?

Reply:
Yeah, I think so. Or, if you want to stick with WS-names, something like `num_slots_per_ws`, or something.